libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -340,6 +340,7 @@ string IRGenerator::generateFunction(FunctionDefinition const& _function)
 	return m_context.functionCollector().createFunction(functionName, [&]() {
 		m_context.resetLocalVariables();
 		Whiskers t(R"(
+			/// @ast-id <astID>
```

Comment:
What about `generateConstructors()`? Don't we want AST ID for the functions it generates?